src/components/tag-input/tag-input.component.html

```diff
@@ -38,12 +38,12 @@
 </ol>
 
 <!-- Insert the custom icon if provided -->
-<div class="ux-tag-icons" *ngIf="icon || _showClearButton">
+<div class="ux-tag-icons" [class.disabled]="disabled" *ngIf="icon || _showClearButton">
```

Comment:
The ux-tag-input element gets the class `disabled` when disabled, so I don't think you need to add another one in here.

Reply:
Updated